src/runtime/lib/API.ts

```diff
@@ -155,6 +156,14 @@ export abstract class AbstractWebView {
 
     public showView(...args: any[]): void {
         if (this._panel === undefined || this._isPanelDisposed) {
+            if (this.shouldPersist === true) {
+                this.options = {
+                    enableScripts: true,
+                    localResourceRoots: this.options.localResourceRoots,
+                    //Add this property to keep the webview persist.
+                    retainContextWhenHidden: true
+                };
```

Comment:
nit: Instead of re-defining these values, you can use the spread operator, like 

```suggestion
                this.options = {
                    ...this.options,
                    //Add this property to keep the webview persist.
                    retainContextWhenHidden: true
                };
```

Reply:
Good point, I'd like to modify as this :)